code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -567,3 +567,15 @@ Contains:
 	armor = list("melee" = 40, "bullet" = 20, "laser" = 20,"energy" = 30, "bomb" = 60, "bio" = 100, "rad" = 90, "fire" = 85, "acid" = 75)
 	slowdown = 0.5
 	w_class = WEIGHT_CLASS_NORMAL
+
+
+/obj/item/clothing/suit/space/scar
```

Comment:
No paired helmet?

Reply:
looks too memorable combined with the helmet.